tex/gloss-marathi.ldf

```diff
@@ -1,91 +1,97 @@
 % TODO implement Hindu calendar (not used in day-to-day practice)
 
 \ProvidesFile{gloss-marathi.ldf}[polyglossia: module for marathi]
 \ifluatex
-  \xpg@warning{Marathi is not supported with LuaTeX.\MessageBreak
+\xpg@warning{Marathi is not supported with LuaTeX.\MessageBreak
 I will proceed with the compilation, but\MessageBreak
 the output is not guaranteed to be correct\MessageBreak
 and may look very wrong.}
 \fi
 \RequirePackage{devanagaridigits}
 
 \PolyglossiaSetup{marathi}{
-  bcp47=mr,
-  script=Devanagari,
-  scripttag=deva,
-  langtag=MAR,
-  hyphennames={marathi},
-  hyphenmins={2,2},%CHECK
-  fontsetup=true,
-  localnumeral=marathinumerals
-  %TODO nouppercase=true,
+script=Devanagari,
+scripttag=deva,
+langtag=MAR,
+hyphennames={marathi},
+hyphenmins={2,2},%CHECK
+fontsetup=true,
+%TODO nouppercase=true,
+%TODO localnumber=marathinumber
 }
 
-% BCP-47 compliant aliases
-\setlanguagealias*{marathi}{mr}
-
+\renewcommand{\baselinestretch}{1.2}
 \def\tmp@western{Western}
 \newif\ifmarathi@devanagari@numerals
 \marathi@devanagari@numeralstrue
 
 \define@key{marathi}{numerals}[Devanagari]{%
-  \def\@tmpa{#1}%
-  \ifx\@tmpa\tmp@western
-    \marathi@devanagari@numeralsfalse
-  \fi}
-
-\newcommand{\marathinumerals}[2]{\marathinumber{#2}}
+\def\@tmpa{#1}%
+\ifx\@tmpa\tmp@western
+\marathi@devanagari@numeralsfalse
+\fi}
 
 \def\marathinumber#1{%
-  \ifmarathi@devanagari@numerals
-    \devanagaridigits{\number#1}%
-  \else
-    \number#1%
-  \fi}
+\ifmarathi@devanagari@numerals
+\devanagaridigits{\number#1}%
+\else
+\number#1%
+\fi}
 
 \def\captionsmarathi{%
-   \def\refname{संदर्भ}%
-   \def\abstractname{सारांश}%
-   \def\bibname{संदर्भ ग्रंथांची यादी}%
-   \def\prefacename{प्रस्तावना}%
-   \def\chaptername{प्रकरण}%
-   \def\appendixname{परिशिष्ट}%
-   \def\contentsname{अनुक्रमणिका}%
-   \def\listfigurename{आकृत्यांची यादी}%
-   \def\listtablename{कोष्टकांची यादी}%
-   \def\indexname{सूची}%
-   \def\figurename{आकृती}%
-   \def\tablename{कोष्टक}%
-   %\def\thepart{}% TODO
-   \def\partname{भाग}%
-   \def\pagename{पान}%
-   \def\seename{पहा}%
-   \def\alsoname{हे सुध्दा पहा}%
-   \def\enclname{समाविष्ट}%
-   \def\ccname{सि.सि.}%
-   \def\headtoname{प्रति}%
-   \def\proofname{सिद्धता}%
-   \def\glossaryname{स्पष्टीकरणांची यादी}%
-   \def\authorsand{आणि}%
+\def\refname{संदर्भ}%
+\def\abstractname{सारांश}%
+\def\bibname{संदर्भसूची}%
+\def\prefacename{प्रस्तावना}%
+\def\chaptername{प्रकरण}%
+\def\appendixname{परिशिष्ट}%
+\def\contentsname{अनुक्रमणिका}%
+\def\listfigurename{आकृत्यांची सूची}%
+\def\listtablename{कोष्टक सूची}%
+\def\indexname{सूची}%
+\def\figurename{आकृती}%
+\def\tablename{कोष्टक}%
+%\def\thepart{}% TODO
+\def\partname{खंड}%
+\def\pagename{पृष्ठ}%
+\def\seename{पाहा}%
+\def\alsoname{हेदेखील पाहा}%
+\def\enclname{समाविष्ट}%
+\def\ccname{सी.सी.}%
+\def\headtoname{प्रती}%
+\def\proofname{सिद्धता}%
+\def\glossaryname{संक्षेपसूची}%
+\def\appendixname{परिशिष्ट}
 }
```

Comment:
Please keep the indentation as in the original

Reply:
Is it right now?